netbox/templates/core/plugin.html

```diff
@@ -0,0 +1,92 @@
+{% extends 'generic/object.html' %}
+{% load helpers %}
+{% load form_helpers %}
+{% load i18n %}
+
+{% block title %}{{ plugin.name }}{% endblock %}
+
+{% block object_identifier %}
+{% endblock object_identifier %}
+
+{% block breadcrumbs %}
+  <li class="breadcrumb-item"><a href="{% url 'core:plugin_list' %}">{% trans "Plugins" %}</a></li>
+{% endblock breadcrumbs %}
+
+{% block subtitle %}
+  <div class="text-secondary fs-5">
+    {{ plugin.tag_line }}
+  </div>
+{% endblock subtitle %}
+
+{% block controls %}
+<div class="text-secondary">
+From
+</div>
+{{ plugin.author }}
+{% endblock %}
+
+{% block tabs %}
+  <ul class="nav nav-tabs" role="tablist">
+    <li class="nav-item" role="presentation">
+      <a class="nav-link active" id="overview-tab" data-bs-toggle="tab" data-bs-target="#overview" type="button" role="tab" aria-controls="edit-form" aria-selected="true">
+        {% trans "Overview" %}
+      </a>
+    </li>
+    {% if not plugin.is_local %}
+    <li class="nav-item" role="presentation">
+      <button class="nav-link" id="version-history-tab" data-bs-toggle="tab" data-bs-target="#version-history" type="button" role="tab" aria-controls="object-list" aria-selected="false">
+        {% trans "Version history" %}
```

Comment:
Seems unnecessary to split this onto a separate tab. We can easily show both the plugin's description and version history on the same page.

Reply:
I'd like to keep the version history on its own tab for now, so that we can experiment with longer and richer content in the Overview tab.